src/xmipp/libraries/data/micrograph.cpp

```diff
@@ -58,6 +58,47 @@ Micrograph::~Micrograph()
     delete IUInt;
     delete IFloat;
 }
+
+Micrograph & Micrograph::operator=(const Micrograph &other)
```

Comment:
this is very error-prone solution. I assume that the complaint was regarding the `Image` pointers. In that case, the better solution would be to use some smart pointer, e.g. `std::unique_ptr`

Reply:
This is a major rewriting. It should wait for the future.